cmd/roll/nodes.go

```diff
@@ -10,19 +10,21 @@ import (
 func nodesCommand(kubectl *kubernetes.Client, verbose *bool) *cobra.Command {
 	var instanceGroup string
 	var cluster string
+	var awsRegion string
 
 	c := &cobra.Command{
 		Use:   "nodes",
 		Short: "",
 		Long:  "",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return roll.Nodes(kubectl, instanceGroup, cluster, *verbose)
+			return roll.Nodes(kubectl, instanceGroup, cluster, awsRegion, *verbose)
 		},
 	}
 	c.Flags().StringVar(&instanceGroup, "kops-instance-group", "", "kops instance group to perfrom the rolling on")
 	c.MarkFlagRequired("kops-instance-group")
 	c.Flags().StringVar(&label, "label", "", "label of the nodes to be rolled")
 	c.Flags().StringVar(&cluster, "cluster", "", "the name of the kops cluster")
+	c.Flags().StringVar(&awsRegion, "aws-region", "us-west-1", "the region to use")
```

Comment:
This should be `eu-west-1` to no break backwards compatibility for now.
```suggestion
	c.Flags().StringVar(&awsRegion, "aws-region", "ue-west-1", "the region to use")
```